doc/tasks.rst

```diff
@@ -596,6 +596,32 @@ You can control the verbosity by:
 * from command line, see :ref:`verbosity option<verbosity_option>`.
 
 
+delayed execution
+-----------------
+
+For :ref:`parallel execution <parallel-execution>` it can be useful to delay
+the start of the first task executed in a process. This can be achieved with
+the ``delay`` option:
+
+.. literalinclude:: tutorial/subtasks-delay.py
```

Comment:
you forgot to commit this file.

Reply:
oops, it must be on my other laptop, I hope I can find it back :confused: